lib/core/src/display/shape/glyph/system.rs

```diff
@@ -108,10 +109,36 @@ impl Line {
             glyph.mod_position(|pos| { *pos += Vector3::new(offset.x,offset.y,0.0); });
             glyph.size().set(size);
         }
+        for glyph in self.glyphs.iter_mut().skip(chars_count) {
+            glyph.size().set(Vector2::new(0.0,0.0));
+        }
+    }
+
+    /// Set the baseline start point for this line.
+    pub fn set_baseline_start(&mut self, new_start:Vector2<f32>) {
+        let offset = new_start - self.baseline_start;
+        for glyph in self.glyphs.iter_mut() {
+            glyph.mod_position(|pos| *pos += Vector3::new(offset.x,offset.y,0.0));
+        }
+        self.baseline_start = new_start;
     }
 }
 
 
+// === Getters ===
+
+impl Line {
+    /// Baseline starting point.
+    pub fn baseline_start(&self) -> &Vector2<f32> { &self.baseline_start }
+    /// Line's height.
+    pub fn height        (&self) -> f32           { self.height          }
+    /// Number of glyphs, giving the maximum length of displayed line.
+    pub fn length        (&self) -> usize         { self.glyphs.len()    }
+    /// Font id.
+    pub fn font_id       (&self) -> FontId        { self.font_id         }
+}
```

Comment:
These docs do not introduce any information. `Line.height` is obviously a line's height. In suhc case I would use `allow(missing_docs)`.

Reply:
I remove some, and some I extended to provide more info.